code/atom.dm

```diff
@@ -3,6 +3,9 @@
 #define DESERIALIZE_NEED_POSTPROCESS 2
 #define DESERIALIZE_NOT_IMPLEMENTED 4
 
+/atom
+	plane = PLANE_DEFAULT
+
```

Comment:
```suggestion
/atom/plane = PLANE_DEFAULT
```

Reply:
Why not make atom.dm contain actual definition of /atom?

Reply:
I think it's defined later in the file already??

Reply:
Oh no, I must have missed that somehow, I'll move it.

Reply:
Done.